jukito/src/main/java/org/jukito/JukitoRunner.java

```diff
@@ -175,19 +167,61 @@ protected void configureTest() {
         return modules;
     }
 
-    private JukitoModule createJukitoModule(final Iterable<Class<? extends Module>> moduleClasses) {
-        return new JukitoModule() {
-            @Override
-            protected void configureTest() {
-                for (Class<? extends Module> mClass : moduleClasses) {
-                    try {
-                        install(mClass.newInstance());
-                    } catch (Exception e) {
-                        throw new RuntimeException(e);
-                    }
+    private TestModule getInnerClassModule(final Class<?> testClass) throws InstantiationException, IllegalAccessException {
+        Class<?> currentClass = testClass;
+        while(currentClass != null) {
+            for (final Class<?> innerClass : currentClass.getDeclaredClasses()) {
+                if (TestModule.class.isAssignableFrom(innerClass)) {
+                    return (TestModule) innerClass.newInstance();
                 }
             }
-        };
+            currentClass = currentClass.getSuperclass();
+        }
+        return null;
+    }
+
+    private boolean getAutoBindMocksValue(final Class<?> testClass) {
+        boolean autoBindMocks = true;
+        Class<?> currentClass = testClass;
+        while (currentClass != null) {
+            final UseModules useModules = currentClass.getAnnotation(UseModules.class);
+            if (useModules != null) {
+                autoBindMocks = useModules.autoBindMocks();
+                break;
+            }
+            currentClass = currentClass.getSuperclass();
+        }
+        return autoBindMocks;
+    }
+
+    private TestModule createJukitoModule(final Iterable<Class<? extends Module>> moduleClasses, final boolean autoBindMocks) {
+        if (autoBindMocks) {
+            return new JukitoModule() {
+                @Override
+                protected void configureTest() {
+                    for (Class<? extends Module> mClass : moduleClasses) {
+                        try {
+                            install(mClass.newInstance());
+                        } catch (Exception e) {
+                            throw new RuntimeException(e);
```

Comment:
When could install fail? This could hide the problem, it may be useful to throw a meaningful exception.